app/src/sharedTest/java/org/oppia/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -90,94 +120,200 @@ class InputInteractionViewTestActivityTest {
 
   @Test
   fun testFractionInputInteractionView_withInputtedWholeNumberValue_hasCorrectPendingAnswer() {
-    val activityScenario = ActivityScenario.launch(InputInteractionViewTestActivity::class.java)
-    onView(withId(R.id.test_fraction_input_interaction_view)).perform(typeText("5 9/10"))
+    val activityScenario = ActivityScenario.launch(
+      InputInteractionViewTestActivity::class.java
+    )
+    onView(withId(R.id.test_fraction_input_interaction_view))
+      .perform(
+        typeText(
+          "5 9/10"
+        )
+      )
     activityScenario.onActivity { activity ->
       val pendingAnswer = activity.fractionInteractionViewModel.getPendingAnswer()
       assertThat(pendingAnswer.answer).isInstanceOf(InteractionObject::class.java)
-      assertThat(pendingAnswer.answer.objectTypeCase).isEqualTo(InteractionObject.ObjectTypeCase.FRACTION)
+      assertThat(pendingAnswer.answer.objectTypeCase).isEqualTo(
+        InteractionObject.ObjectTypeCase.FRACTION
+      )
       assertThat(pendingAnswer.answer.fraction.isNegative).isEqualTo(false)
       assertThat(pendingAnswer.answer.fraction.wholeNumber).isEqualTo(5)
       assertThat(pendingAnswer.answer.fraction.numerator).isEqualTo(9)
       assertThat(pendingAnswer.answer.fraction.denominator).isEqualTo(10)
     }
   }
 
+  /* ktlint-disable max-line-length */
   @Test
   fun testFractionInputInteractionView_withInputtedNegativeWholeNumberValue_hasCorrectPendingAnswer() {
-    val activityScenario = ActivityScenario.launch(InputInteractionViewTestActivity::class.java)
-    onView(withId(R.id.test_fraction_input_interaction_view)).perform(typeText("-55 59/9"))
+    val activityScenario = ActivityScenario.launch(
+      InputInteractionViewTestActivity::class.java
+    )
+    onView(withId(R.id.test_fraction_input_interaction_view))
+      .perform(
+        typeText(
+          "-55 59/9"
+        )
+      )
     activityScenario.onActivity { activity ->
       val pendingAnswer = activity.fractionInteractionViewModel.getPendingAnswer()
       assertThat(pendingAnswer.answer).isInstanceOf(InteractionObject::class.java)
-      assertThat(pendingAnswer.answer.objectTypeCase).isEqualTo(InteractionObject.ObjectTypeCase.FRACTION)
+      assertThat(pendingAnswer.answer.objectTypeCase).isEqualTo(
+        InteractionObject.ObjectTypeCase.FRACTION
+      )
       assertThat(pendingAnswer.answer.fraction.isNegative).isEqualTo(true)
       assertThat(pendingAnswer.answer.fraction.wholeNumber).isEqualTo(55)
       assertThat(pendingAnswer.answer.fraction.numerator).isEqualTo(59)
       assertThat(pendingAnswer.answer.fraction.denominator).isEqualTo(9)
     }
   }
+  /* ktlint-enable max-line-length */
```

Comment:
Same comment here, we should disable only up till the function name, not the entire function block

Reply:
Done